pelican/plugins/seo/tests/test_html_content.py

```diff
@@ -0,0 +1,40 @@
+""" Units tests for SEO Enhancer. """
```

Comment:
I guess that with my previous comment, you don't need to add a test. All the tests in [this file](https://github.com/pelican-plugins/seo/blob/master/pelican/plugins/seo/tests/test_seo_enhancer.py) you'll have to update will test if no whitespace is inserted.

Reply:
Removed the new test file and updated test cases in _test_seo_enhancer.py_. I used BeautifulSoup to format the output so the tests look more readable but added additional asserts on raw output.

Reply:
I was going for the minimal change but I agree that reformatting the HTML is the right thing to do. Fixed.